spec/factories/schedules.rb

```diff
@@ -0,0 +1,18 @@
+FactoryGirl.define do
+  factory :schedule do
+    # Hackney
+    location_id 'ac7112c3-e3cf-45cd-a8ff-9ba827b8e7ef'
+
+    trait :blank do
+      begin
+        Schedule.slot_attributes.each do |attribute|
+          add_attribute(attribute, false)
+        end
+      rescue ActiveRecord::StatementInvalid => e
```

Comment:
This happens during migrations (though totally benign) since `Schedule#slot_attributes` causes a database read and the factories are evaluated early on.